workers/config/src/test/kotlin/ConfigWorkerTest.kt

```diff
@@ -274,9 +274,8 @@ private fun mockValidator(result: ConfigValidationResult): ConfigValidator {
  * Create a mock [ConfigManager]. The mock is configured to return the test validation script.
  */
 private fun mockConfigManager(): ConfigManager = mockk<ConfigManager> {
-    every {
-        getFileAsString(Context(RESOLVED_CONTEXT), ConfigWorker.VALIDATION_SCRIPT_PATH)
-    } returns PARAMETERS_SCRIPT
+    every { containsFile(Context(RESOLVED_CONTEXT), ConfigWorker.VALIDATION_SCRIPT_PATH) } returns true
```

Comment:
There should be a separate test case for this scenario.

Reply:
I've added a separate test for a missing validation script, but I cannot get it fully to run:

    Verification failed: call 3 of 4: OrtRun(child of #81#82).getJobConfigs()) was not called

What confuses me is that there seems to be no verification for `getJobConfigs()` implemented...

Reply:
I tested it locally and found the problem: In `verify`, you do

```
                ortRunRepository.update(
                    id = RUN_ID,
                    resolvedJobConfigs = context.ortRun.jobConfigs.asPresent(),
                    resolvedJobConfigContext = RESOLVED_CONTEXT.asPresent()
                )
```
The problem is that `context` is a mock as well, so it tries to verify the invocations on this object, too. What worked for me, was obtaining the job configurations before the `verify` block, e.g.
`val expectedJobConfigs = context.ortRun.jobConfigs`

Then, in the `verify` block, use this variable.